.github/workflows/cypress.yml

```diff
@@ -102,7 +102,17 @@ jobs:
         run: npx @wordpress/env@latest start
 
       - name: Run Cypress Tests
-        run: npm run test:e2e
+        uses: replayio/action-cypress@v0.2.2
+        with:
+          api-key: ${{ secrets.RECORD_REPLAY_API_KEY }}
+          issue-number: ${{ github.event.pull_request.number }}
+          browser: 'Replay Chromium'
+          command: npm run test:e2e --
+          upload-all: true
```

Comment:
In case this isn't clear, you can omit this if you only want to upload failed tests. It can be useful to upload everything to compare failures with previous successful runs but that does come at a runtime cost. Alternatively, you can run a nightly job that uploads everything so you still have a baseline for comparison.